src/envoy/token/mocks.h

```diff
@@ -28,13 +28,16 @@ class MockTokenSubscriberFactory : public TokenSubscriberFactory {
  public:
   MOCK_METHOD(TokenSubscriberPtr, createImdsTokenSubscriber,
               (const TokenType& token_type, const std::string& token_cluster,
-               const std::string& token_url, UpdateTokenCallback callback),
+               const std::string& token_url,
+               const std::chrono::seconds fetch_timeout,
+               UpdateTokenCallback callback),
               (const));
 
   MOCK_METHOD(
       TokenSubscriberPtr, createIamTokenSubscriber,
       (const TokenType& token_type, const std::string& token_cluster,
-       const std::string& token_url, UpdateTokenCallback callback,
+       const std::string& token_url, const std::chrono::seconds fetch_timeout,
```

Comment:
nit:  const std::chrono::seconds&    
add &,  otherwise, it doesn't make sense

Reply:
If it is a small struct,  like string_view,   it is better to copy the value so remove "const &"
I believe std::chrono::seconds is a very small struct.

Reply:
I took a look. I'd prefer to do the change to `absl::string_view` in a different PR. A lot of the files in /src/envoy/token/ use `const std::string&`, and the diff is too big already.

My original code for `std::chrono::seconds` seems correct. 
- Pass `std::chrono::seconds` by value since it just wraps a single variable. No need for reference. 
- Still keep it as `const` because it's not immutable (unlike `string_view`). It has a `++` / `--` operator that we should guard against.

Reply:
Sorry,  my requirement is just to remove "const"  for std::chrono::seconds.   I used string_view just as an example.  

Here is my understanding
foo(const T t)  doesn't make sense since t is copied into the function.  It is exactly the same as foo(T t). 

const T& t makes sense, it is referencing

Reply:
I see, done.